cannon/mipsevm/tests/fuzz_evm_common64_test.go

```diff
@@ -0,0 +1,54 @@
+//go:build cannon64
+// +build cannon64
+
+package tests
+
+import (
+	"math/big"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/cannon/mipsevm/arch"
+	"github.com/ethereum-optimism/optimism/cannon/mipsevm/testutil"
+)
+
+func FuzzStateDmultInsn(f *testing.F) {
+	versions := GetMipsVersionTestCases(f)
+	f.Fuzz(func(t *testing.T, rs arch.Word, rt arch.Word, seed int64) {
```

Comment:
If I change the param type to a signed int, I start to get test failure here:
```suggestion
	f.Fuzz(func(t *testing.T, rs int64, rt int64, seed int64) {
```

When running this locally with the existing `Word` (unsigned) operands, the fuzzer seems to only be generating 1 byte values between 0 and 255.